--- origsrc/libshout-2.3.1/examples/Makefile.am	2012-02-01 14:59:15.000000000 -0600
+++ src/libshout-2.3.1/examples/Makefile.am	2013-11-20 01:03:33.559719500 -0600
@@ -11,4 +11,4 @@ nonblocking_SOURCES = nonblocking.c
 nonblocking_LDADD = $(top_builddir)/src/libshout.la @SHOUT_LIBDEPS@
 
 AM_CFLAGS = @XIPH_CFLAGS@
-AM_CPPFLAGS = @XIPH_CPPFLAGS@ -I$(top_builddir)/include
+AM_CPPFLAGS = @XIPH_CPPFLAGS@ -I$(top_builddir)/include -I$(top_srcdir)/include
--- origsrc/libshout-2.3.1/src/Makefile.am	2013-11-19 23:37:41.084014800 -0600
+++ src/libshout-2.3.1/src/Makefile.am	2013-11-19 23:44:27.590265600 -0600
@@ -28,7 +28,7 @@ AM_CFLAGS = @XIPH_CFLAGS@
 libshout_la_LIBADD = net/libicenet.la timing/libicetiming.la avl/libiceavl.la\
 		httpp/libicehttpp.la $(MAYBE_THREAD_LIB) $(THEORA_LIBS) $(VORBIS_LIBS) $(SPEEX_LIBS)
 
-INCLUDES = -I$(top_builddir)/include
+INCLUDES = -I$(top_builddir)/include -I$(top_srcdir)/include
 
 debug:
 	$(MAKE) all CFLAGS="@DEBUG@"
